server/views/pages/search/_searchResults.njk

```diff
@@ -0,0 +1,23 @@
+{% set prisoners = searchResults.value.prisoners %}
+{% set pagination = searchResults.value.pagination %}
+
+<table class="govuk-table" data-qa="search-results-table">
+  <caption><span class="govuk-visually-hidden">column headers with buttons are sortable.</span></caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row" data-qa="sortable-table-headers">
+      <th scope="col" class="govuk-table__header">Prisoner Name</th>
```

Comment:
The column headings are not sortable yet, but they will be soon ....
(this PR is about introducing the Result class and pattern, rather than a finished UI)